apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -214,6 +219,17 @@ export default function ListBox({
 
   const isVertical = listLayout !== 'horizontal';
   const count = layout.qListObject.qSize.qcy;
+
+  const getCalculatedHeight = (ps) => {
+    // If values have been filtered in the currently loaded page, we want to
+    // prevent rendering empty rows by assigning the actual number of items to render
+    // since count (qcy) does not reflect this in DQ mode currently.
+    const hasFilteredValues = ps.some((page) => page.qArea.qHeight < MINIMUM_BATCH_SIZE);
+    const h = ps.reduce((tot, page) => tot + page.qArea.qHeight, 0);
+    return hasFilteredValues ? h : count;
+  };
```

Comment:
Can we use `qTop` to account for rows prior to the ones just loaded:
```
const nbrOfPriorRows = pages[0].qArea.qTop;
const h = nbrOfPriorRows + ps.reduce((tot, page) => tot + page.qArea.qHeight, 0);
```
?

Reply:
🤔 💭

Reply:
Try `const h = Math.max(...ps.map((page) => page.qArea.qTop + page.qArea.qHeight));`